st2common/tests/unit/test_crypto_utils.py

```diff
@@ -53,6 +54,31 @@ def setUpClass(cls):
         super(CryptoUtilsTestCase, cls).setUpClass()
         CryptoUtilsTestCase.test_crypto_key = AESKey.generate()
 
+    def test_symmetric_encrypt_decrypt_short_string_needs_to_be_padded(self):
+        original = u'a'
+        crypto = symmetric_encrypt(CryptoUtilsTestCase.test_crypto_key, original)
+        plain = symmetric_decrypt(CryptoUtilsTestCase.test_crypto_key, crypto)
+        self.assertEqual(plain, original)
+
+    def test_symmetric_encrypt_decrypt_utf8_character(self):
+        values = [
+            u'£',
```

Comment:
I don't think the `u` syntax is supported in Python 3. Better to use [`six.u()`](https://pythonhosted.org/six/#six.u) so we can run our tests with Python 3.

Reply:
It is - `u` as in prefix for strings which is also a default (e.g. `u'foo`).

Those tests also run under Python 2.x and 3.6 on Travis and I also ran them under both versions locally.

Reply:
Awesome, good to know.